merlin/examples/dev_workflows/minimum_format.yaml

```diff
@@ -0,0 +1,9 @@
+description:
```

Comment:
This is good to test the other sections, but we should make sure merlin fails correctly if there is no description or study sections.

Reply:
Added